specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/examples/Certificate_CreateOrUpdate.json

```diff
@@ -0,0 +1,29 @@
+{
+  "parameters" :{
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "examplerg",
+    "managedEnvironmentName": "testcontainerenv",
+    "name": "certificate-firendly-name",
+    "api-version": "2022-01-01-preview",
+    "certificateEnvelope": {
+        "password":"private key password",
+        "name":"certificate-firendly-name",
+        "value" :"PFX-or-PEM-blob"
+    }
+  },
+  "responses": {
+    "200" :{
+      "certificate": {
```

Comment:
This should be in a standard ARM resource envelope, e.g

{
  "id" : "arm id"
  "name": unique name
  "type": "Microsoft.App/ManagedEnvironments/Certificates"
  "properties": {
     cert specific properties here
 }
}

Reply:
Yes. Made the change